Sources/AppsFlyerInstance.swift

```diff
@@ -86,6 +101,28 @@ public class AppsFlyerInstance: NSObject, AppsFlyerCommand {
         if let customData = settings[AppsFlyerConstants.Settings.customData] as? [AnyHashable: Any] {
             appsFlyer.customData = customData
         }
+        if let disableAppleAdsAttribution = settings[AppsFlyerConstants.Settings.disableAppleAdsAttribution] as? Bool {
+            appsFlyer.disableAppleAdsAttribution = disableAppleAdsAttribution
+        }
+        if let enableTCFDataCollection = settings[AppsFlyerConstants.Settings.enableTCFDataCollection] as? Bool {
+            appsFlyer.enableTCFDataCollection(enableTCFDataCollection)
+        }
+        if let appInviteOneLinkID = settings[AppsFlyerConstants.Settings.appInviteOneLinkID] as? String {
+            appsFlyer.appInviteOneLinkID = appInviteOneLinkID
+        }
+        if let deepLinkTimeout = settings[AppsFlyerConstants.Settings.deepLinkTimeout] as? Int {
+            appsFlyer.deepLinkTimeout = UInt(deepLinkTimeout)
+        }
```

Comment:
We should check that deeplink is greater than or equal to 0 or this could crash
```suggestion
        if let deepLinkTimeout = settings[AppsFlyerConstants.Settings.deepLinkTimeout] as? Int,
           deepLinkTimeout >= 0 {
            appsFlyer.deepLinkTimeout = UInt(deepLinkTimeout)
        }
```

Reply:
hmm, since we talked about the logic should be in VendorRemoteCommand, so I created a similar thing there